11LessonEleven/homework/css/inspired.css

```diff
@@ -0,0 +1,74 @@
+
+
+/* Laptop, Default size  */
+.box{
+  width: 40%;
+  height: 150px;
+  text-align: left;
+  color: black;
+  float: left;
+  padding-left: 5%;
+  background-color: #F7F7F7;
```

Comment:
be careful! if you don't have the opacity, this color will cover the background image of its outer div. instead of a hex code, use rgba(value, value, value, value) to control the opacity with `a`.